rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,497 @@
+
+#### HCA Sample Metadata
+
+The HCA metadata team has built a metadata specification draft for
+imaging data that references data in SpaceTx-Format. The issue that
+formed the draft can be found
+[<span class="underline">here</span>](https://github.com/HumanCellAtlas/metadata-schema/issues/368).
+
+### Data and Metadata validation
+
+#### Input Data validation
+
+SpaceTx Format consists of a json file that specifies how to compose a
+dataset from a set of two-dimensional images. As a result, it is
+possible to rapidly validate that a file is complete and internally
+consistent. In addition, SpaceTx Format stores per-chunk checksum
+information, enabling the data to be checked for integrity. It is
+possible to carry out additional, more data-specific checks, such as
+that images are not blank, contain spots, or contain cells, however
+these checks take increasing amounts of time.
+
+OME and SpaceTx can provide a docker container that validates image
+datasets with the highest level of stringency possible in a DCP-provided
+"maximum validation time".
+
+#### Output Data validation
+
+Each of the output formats produced by SpaceTx can be stored in Zarr and
+read (on-demand) by *starfish* and xarray. SpaceTx provides a docker
+container that loads and validates each output file using the xarray
+library. Validation can be tuned to occur within a DCP-provided target
+validation time.
+
+### Data Processing Workflows
+
+To process data in the DCP, the *starfish* team can deliver WDL-wrapped
+data processing workflows for any of the existing image-based
+transcriptomics pipelines. These workflows are based on the *starfish*
+python library, which in turn leverages standard open-source python
+projects. *starfish* provides a docker container that exposes both a
+python API and a CLI, and supports composable, modular, pipelines. These
+pipelines take as inputs images in SpaceTx Format and a pipeline recipe.
+Each assay emits output in the formats specified above. More detail on
+the state of image-based transcriptomics analyses can be found
+[<span class="underline">here</span>](https://docs.google.com/document/d/18Ve9C_cEn5lI2paivfgTAyjsM62qMyUF_s4mj82l1L0/edit#).
+
+### Data Processing Infrastructure
+
+All *starfish* pipelines share the same inputs and outputs. It should
+therefore be possible to deliver a single adapter WDL that satisfies any
+SpaceTx assay, and a family of subscriptions with minimal inter-class
+changes that specify the particular assay in question.
+
+### Delivery of Output Data to Users
+
+#### Matrices
+
+As mentioned above, image-based transcriptomics output data adhere to
+existing Matrix Service specifications and can be served "as is" for the
+imaging prototype. Additional information on cellular geometries and QC
+will be provided as a supplementary metadata zarr archive.
+
+However, other imaging experiments will likely require a generalization
+of the internal namespace of the DCP. Currently, the DCP standardizes
```

Comment:
Using "internal namespace of the DCP" makes me feel like you are talking about domain spaces and is confusing. Maybe "will likely requiring extending how features (genes) are named to a support a broader set of features (eg. proteins and cellular components).

Reply:
@josh can you include Tim's wording in the next pass?